// 
try {
  //  код, который может выбросить ошибку

} catch (error) {
  //  код для обработки ошибки

}

try {
  //  код, который может выбросить ошибку
} catch (error) {
  //  код для обработки ошибки
} finally {
  //  код, который выполнится в любом случае
}

try {
  //  код, который может выбросить ошибку
} catch (error) {
  //  код для обработки ошибки
  throw new Error('Ошибка в блоке catch');
}